src/home/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/index.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="toolbar">
+      <a href="#">My Logo</a>
+      <nav>
+        <img src="../assert/images/toggle.png" alt="Toggle" />
+      </nav>
+    </header>
+    <section class="headline">
+      <p class="heading">I am Mwafrika Glad to see you!</p>
+      <p class="description">
+        I’m a software developer! I can help you build a product , feature or

```

Comment:
- [ ] [OPTIONAL] The text `I'm a software developer!` should be on its own tag like `h2` because it is a sub-title. And you should place it above the `p` tag.
- [ ] [OPTIONAL] Secondly, ensure you use the correct fonts and font sizes given from your chosen Figma template. Your content currently, doesn't make use of the correct fonts.

Reply:
I am done with all the feedback. please review so that I can merge